src/codeu/chat/common/NetworkCode.java

```diff
@@ -35,5 +35,10 @@
       RELAY_READ_REQUEST = 27,
       RELAY_READ_RESPONSE = 28,
       RELAY_WRITE_REQUEST = 29,
-      RELAY_WRITE_RESPONSE = 30;
+      RELAY_WRITE_RESPONSE = 30,
+      SERVER_UPTIME_REQUEST = 31,
+      SERVER_UPTIME_RESPONSE = 32;
```

Comment:
@lysanderp41 it was my fault, when I merged i forgot to change it to a comma instead of a semicolon, @joecarnahan is right is should be a comma instead of a semicolon. I think i fixed it in the most recent commit!